app/controllers/message_drafts_controller.rb

```diff
@@ -8,6 +8,8 @@ class MessageDraftsController < ApplicationController
   include MessageThreadsConcern
 
   def index
+    redirect_to message_threads_path(q: "label:(#{Current.tenant.draft_tag.name})") unless Current.tenant.feature_enabled?(:message_draft_import)
+
```

Comment:
Takto nejak @jsuchal?

Reply:
Mozno by som vytvoril before_filter :ensure_drafts_enabled a tam hodit redirect ak to nie je zapnute